src/hooks/useOpenModal.js

```diff
@@ -0,0 +1,10 @@
+import { useState } from "react";
+
+export default function useOpenModal() {
```

Comment:
should be refactored:
* hook name is very common and suppose to manage with modal state, but instead we have isLogin, isRegister
*